companies/landespolizei-des-fuerstentums-liechtenstein.json

```diff
@@ -0,0 +1,18 @@
+{
+    "slug": "landespolizei-des-fuerstentums-liechtenstein",
+    "relevant-countries": [
+        "li"
+    ],
+    "categories": [
+        "public body"
+    ],
+    "name": "Landespolizei des Fürstentums Liechtenstein",
+    "address": "Gewerbeweg 4\nPostfach 684\n9490 Vaduz\nLiechtenstein",
+    "phone": "+423 236 71 11",
+    "email": "info@landespolizei.li",
```

Comment:
Considering the Regierungskanzlei is mentioned as the data protection officer in the privacy policy, I think we should use the `Datenschutz@regierung.li` email.